pwndbg/dbg/lldb/repl/__init__.py

```diff
@@ -484,18 +484,24 @@ def start_ipi():
 target_create_ap.add_argument("-v", "--version")
 target_create_ap.add_argument("filename")
 target_create_unsupported = [
-    "sysroot",
-    "arch",
     "build",
     "core",
     "no-dependents",
-    "platform",
     "remote-file",
     "symfile",
     "version",
 ]
 
 
+def _get_target_triple(debugger: lldb.SBDebugger, filepath: str) -> str | None:
+    target: lldb.SBTarget = debugger.CreateTarget(filepath)
```

Comment:
Pls add a docstring with examples what a target triple is, what it can be (e.g. `"a-b-c"` etc).

Reply:
fixed